_posts/2022-03-25-dyadic-fairness.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "On Dyadic Fairness: Exploring and Mitigating Bias in Graph Connections"
+tags: [graphs, fairness, representation learning]  # This should be the relevant areas related to your blog post
+authors: Arjun Subramonian
+---
+
+![The image contains two side-by-side panels. The left panel contains a social network of people. Their names are Adam, David, Maria, Maya, and Sophia. There are undirected edges between the people: (Adam, Maria), (David, Maria), (David, Maya), (Maria, Maya), (Maria, Sophia), (Maya, Sophia), represented by solid lines. The right panel contains the same social network with the addition of the edges (Adam, Sophia) and (David, Sophia), represented by dashed lines. The solid lines indicate the observed edges passed as input to a link prediction algorithm and the dashed lines indicate the edges predicted by the algorithm.]({{ site.url }}/public/images/2021-12-01-dyadic-fairness/link_prediction.png)
+<center>[<b>Figure 1, Source:</b> <a href="https://www.nature.com/articles/s41598-019-57304-y">https://www.nature.com/articles/s41598-019-57304-y</a>]</center>
+\\
+This blog post discusses the **ICLR 2021 paper ["On Dyadic Fairness: Exploring and Mitigating Bias in Graph Connections"](https://openreview.net/forum?id=xgGS6PmzNq6) by Li et al.**, **highlighting the importance of its theoretical results** while **critically examining the notions and applications of dyadic fairness** presented. This blog post assumes basic familiarity with graph representation learning using message-passing GNNs and fairness based on observed characteristics. The images in this blog post are equipped with **alernative text**.
```

Comment:
Could you move this paragraph above the first image? This will then be taken as a nice excerpt for the `blog/` endpoint:

![image](https://user-images.githubusercontent.com/19653980/163635958-64eb4cd8-f381-448c-b498-17c7eaf42d03.png)